src/main/java/it/gov/pagopa/afm/calculator/service/CalculatorService.java

```diff
@@ -173,14 +173,14 @@ private Transfer createTransfer(
         .idChannel(bundle.getIdChannel())
         .onUs(this.getOnUsValue(taxPayerFee, bundle, paymentOption))
         .abi(bundle.getAbi())
+        .subThreshold(isSubThreshold(taxPayerFee))
```

Comment:
use `payment amount` instead of `taxPayerFee`

Reply:
done but the threshold has been moved as a global variable